src/scripts/main.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
 
+const htmlBody = document.body;
+
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const errorBlock = document.createElement('div');
```

Comment:
it might be an error block or a successful one, or warning. So it's better to make more abstract name
```suggestion
  const notificationBlock = document.createElement('div');
```